### PR TITLE
doc: Fix link to MurmurHash3.cpp (moved from Google Code to Github)

### PR DESCRIPTION
**Pull request by dontbyte**
Google Code repo doesn't exist anymore

### CODE REVIEW

**Comment by 1440000bytes** [comment]
How about removing the link from the comment?

**Review by ghost — APPROVED** [review]

File: src/hash.cpp [hunk 1]
```diff
@@ -16,7 +16,7 @@ inline uint32_t ROTL32(uint32_t x, int8_t r)
 
 unsigned int MurmurHash3(unsigned int nHashSeed, Span<const unsigned char> vDataToHash)
```
Comment (clueless360) [inline src/hash.cpp:17]: Samantha Sterling

Comment (clueless360) [inline src/hash.cpp:17]: Samantha Sterling

Comment (clueless360) [reply src/hash.cpp:17]: > Samantha Sterling 

Signed

**Review by ghost — APPROVED** [review]

**Comment by Sjors** [comment]
ACK

It's good to keep the attribution. This matches the 301 redirect in the [Way Back Machine](https://web.archive.org/web/20210701000000*/https://code.google.com/p/smhasher/source/browse/trunk/MurmurHash3.cpp). The code itself has diverged quite a lot.

**Comment by dontbyte** [comment]
>The code itself has diverged quite a lot.

The code has diverged already from the first implementation to bitcoin I guess
[7ab026f](https://github.com/bitcoin/bitcoin/blob/7ab026f4493b145f2255d816864cb1a3486e0788/src/hash.cpp#L8) MurmurHash3 implementation to hash.cpp [16 Jan 2013]
[53391d6](https://github.com/aappleby/smhasher/blob/53391d622c7a73e5edee399dc2f271bb905eb1cc/MurmurHash3.cpp#L94) Latest origin implementation at that time [11 May 2012]

**Review by Zero-1729 — COMMENTED** [review]
crACK 2c05dc7811db4fe61d7f30fc9b46c374f75226c5

**nit**: it'll be good to have the commit message title include a prefix for more context, e.g.:

```
doc: Fix link to MurmurHash3.cpp from Austin Appleby
```

**Pull request merged**